src/control/cmd/daos_agent/mgmt_rpc.go

```diff
@@ -125,12 +134,9 @@ func (mod *mgmtModule) handleGetAttachInfo(ctx context.Context, reqb []byte, pid
 
 	numaNode, err := mod.getNUMANode(ctx, pid)
 	if err != nil {
-		mod.log.Errorf("unable to get NUMA node: %s", err.Error())
+		mod.log.Errorf("%s: unable to get NUMA node: %s", client, err)
 		return nil, err
 	}
-
-	mod.log.Debugf("client process NUMA node %d", numaNode)
```

Comment:
It is interesting to know this in the failure case as well. Keep it at trace level?

Reply:
Seems like maybe it'd be better in `mod.getAttachInfo()`? I can look at adding a log entry there.

Reply:
Just added a trace-level log here, as it's simplest and has access to the new client info.